uwblueprint/childrens-aid-society
# Main Dashboard Create New Case

## Notion ticket link

[Main Dashboard Create New Case](https://www.notion.so/uwblueprintexecs/Main-Dashboard-Create-New-Case-15bf0fc9d1404da28d48df617a770d5d?pvs=4)



## Implementation description
* Linking the new case button to the intake page



## Steps to test
1. Click on New Case button on dashboard
2. Verify you are on intake page

<img width="1385" alt="image" src="https://github.com/uwblueprint/childrens-aid-society/assets/69442773/a23a6c8f-ee68-42e1-8f80-1c2e41cf12ca">
<img width="1293" alt="image" src="https://github.com/uwblueprint/childrens-aid-society/assets/69442773/b33c79e0-a6bf-48ac-be7c-7a1625e2a7ac">




## What should reviewers focus on?
* 


## Checklist
